dojo/tools/blackduck/model.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from typing import Set
+
+
+@dataclass
```

Comment:
This is a 3.7 feature. For older versions, I think you don't even declare any of these attributes. You can just make this a dummy class basically.